cmd/gtk/utils.go

```diff
@@ -19,9 +19,25 @@ import (
 	"github.com/pactus-project/pactus/wallet"
 )
 
+// https://stackoverflow.com/questions/3249053/copying-the-text-from-a-gtk-messagedialog
+func updateMessageDialog(dlg *gtk.MessageDialog) {
+	area, err := dlg.GetMessageArea()
```

Comment:
@b00f for address details we added CSS style and function that user can copy value of a field, can't we use it here too?

Reply:
Note sure if it can help there.